DOCUMENTATION.md

```diff
@@ -15,7 +15,6 @@
 	- [Session Bus](#system-bus)
 - [Tips and Tricks](#tips-and-tricks)
 	- [Manually reset Flatseal permissions](#manually-reset-flatseal-permissions)
-	- [Enable custom installations](#enable-custom-installations)
```

Comment:
Wait, what I requested was the opposite. Keep the `Enable custom installations` and remove `Flatpak 1.7.1 or newer` and the other subsection.

Reply:
What do you mean by "other subsection"? Do you mean to remove every subsection except `Enable custom installations`?

Reply:
No, look carefully at the document structure. We need `Enable custom installations` in the TOC, but not its subsections.

Reply:
Like this? https://github.com/tchx84/Flatseal/pull/171/commits/fcff991d078cc3c1954e2a8241e16570fed7fa4d

Reply:
https://github.com/tchx84/Flatseal/pull/171/commits/82ad44c7e91798df02467f50695ef7df2e63166d